src/platform/web/scrollcomponent/ScrollComponent.tsx

```diff
@@ -3,6 +3,8 @@ import { Dimension } from "../../../core/dependencies/LayoutProvider";
 import BaseScrollComponent, { ScrollComponentProps } from "../../../core/scrollcomponent/BaseScrollComponent";
 import BaseScrollView, { ScrollEvent } from "../../../core/scrollcomponent/BaseScrollView";
 import ScrollViewer from "./ScrollViewer";
+import debounce = require("lodash.debounce");
```

Comment:
```suggestion
import debounce from "lodash.debounce";
```

Reply:
Yeah, something funky is going on here and I _think_ that the dynamic require might be necessary. It was like this elsewhere in the library too, but IDK why 🤷